apps/ledger-live-desktop/tests/enum/Currency.ts

```diff
@@ -7,6 +7,8 @@ export class Currency {
     public readonly deviceLabel: string,
     public readonly sendPattern: DeviceLabels[],
     public readonly receivePattern: DeviceLabels[],
+    public readonly tokenName?: string,
```

Comment:
1. do we need deviceLabel in Currency enum if it's always the same as uiName ?
2. also, from my understanding tokenName  should be equal uiName , meaning we don't need extra property tokenName
3. same for uiLabel  === tokenTicker